src/test/fuzz/uint256.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020-2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <uint256.h>
+#include <util/transaction_identifier.h>
+
+FUZZ_TARGET(uint256_fromhex)
```

Comment:
Ah yes, I missed that it's already covered. Do you think it's best to drop the fuzz commit altogether or move the extra checks into `fuzz/hex.cpp`?

<details>
<summary>git diff on 30c7351491</summary>

```diff
diff --git a/src/Makefile.test.include b/src/Makefile.test.include
index 268d346d6b..0993a65eff 100644
--- a/src/Makefile.test.include
+++ b/src/Makefile.test.include
@@ -397,7 +397,6 @@ test_fuzz_fuzz_SOURCES = \
  test/fuzz/tx_pool.cpp \
  test/fuzz/txorphan.cpp \
  test/fuzz/txrequest.cpp \
- test/fuzz/uint256.cpp \
  test/fuzz/utxo_snapshot.cpp \
  test/fuzz/utxo_total_supply.cpp \
  test/fuzz/validation_load_mempool.cpp \
diff --git a/src/test/fuzz/hex.cpp b/src/test/fuzz/hex.cpp
index bc46863f1d..dcf2df279e 100644
--- a/src/test/fuzz/hex.cpp
+++ b/src/test/fuzz/hex.cpp
@@ -27,7 +27,11 @@ FUZZ_TARGET(hex)
         assert(ToLower(random_hex_string) == hex_data);
     }
     (void)IsHexNumber(random_hex_string);
-    (void)uint256::FromHex(random_hex_string);
+    if (uint256::FromHex(random_hex_string)) {
+        assert(random_hex_string.length() == 64);
+        assert(Txid::FromHex(random_hex_string));
+        assert(Wtxid::FromHex(random_hex_string));
+    }
     (void)uint256S(random_hex_string);
     try {
         (void)HexToPubKey(random_hex_string);

```
</details>

I'm not 100% sure on the `assert(random_hex_string.length() == 64);` check since that feels like more of a unittest thing, but at the same time it's pretty cheap check

Reply:
Thanks, I've applied the above diff to fcf7fe35f1805e59357bcb6e65fa66460b423f86